docs/template-helpRequest.md

```diff
@@ -0,0 +1,41 @@
+**Help Request**
+
+**Issue Title:** Help Request: [summary of your problem]
+
+**Step in the `settingUp.md` Guide / Failing Tests**
+
+*If you are setting up the project:*
+> Tell us the step in the setting up guide at which you are stuck
+
+*If you are resolving test failures:*
+> Tell us which tests are failing
+
+**Error Messages Shown...**
+
+*...in the TestNG tab, if you are resolving test failures:*
+> e.g. Failed tests:
+> parallel-ui-tests -> teammates.test.cases.ui.browsertests.InstructorFeedbackResultsPageUITests
```

Comment:
This does not really help. We need something like full stack trace (or screenshot of it).
Pardon the verbosity, but things like this will help the dev team better:
```
java.lang.AssertionError: 
	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
	at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)
	at org.testng.AssertJUnit.assertNotNull(AssertJUnit.java:267)
	at org.testng.AssertJUnit.assertNotNull(AssertJUnit.java:259)
	at teammates.test.cases.BaseTestCase.assertNotNull(BaseTestCase.java:177)
	at teammates.test.cases.action.AdminAccountDeletePageActionTest.testExecuteAndPostProcess(AdminAccountDeletePageActionTest.java:42)
```